webapi/src/main/scala/org/knora/webapi/store/triplestore/upgrade/RepositoryUpdater.scala

```diff
@@ -168,14 +191,15 @@ class RepositoryUpdater(
     val downloadDir: Path = settings.upgradeDownloadDir match {
       case Some(configuredDir) =>
         // Yes. Use that directory.
+        // TODO-mpro never used - see application.conf line 341
```

Comment:
Can this comment be deleted? The content of application.conf can vary from one setup to another, so the reference to a certain line and value in that conf file may not be valid anymore.

Reply:
You know what, actually you had right. It doesn't make sense., because we never get back to it (at least in here).